src/Microsoft.AspNet.Mvc.DataAnnotations/DataAnnotationsModelValidatorProvider.cs

```diff
@@ -51,7 +53,9 @@ public void GetValidators(ModelValidatorProviderContext context)
                     continue;
                 }
 
-                var validator = new DataAnnotationsModelValidator(attribute, stringLocalizer);
+                var validator = new DataAnnotationsModelValidator(
```

Comment:
It doesn't fit in 120, I probably tried, saw that it didn't fit and forgot about it.